magento/inventory
# Create Stock table at the same time stock is created #2569

### Description (*)
This pull request fixes the performance issue for frontend asking if stock table exits 

### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2569

### Manual testing scenarios (*)

1. Create two Sources **Munich** and **Hamburg** 
2. Create new Stock Germany 
3. Go in the Database there should  be a table with the name **inventory_stock_2**
4.  Create Product assigned them to Munich and Hamburg add Stock of 5
5. Go in the Frontend  you see your Product and can add Stock of 10 in the Cart

### Questions or comments


### Contribution checklist (*)
